library/Cargo.toml

```
@@ -43,3 +43,4 @@ unicode-math-class = "0.1"
 unicode-script = "0.5"
 unicode-segmentation = "1"
 xi-unicode = "0.3"
+time = {version = "0.3.20", features = ["formatting"]}
```

```suggestion
time = { version = "0.3.20", features = ["formatting"] }
```
could you put it into the alphabetically correct position?